src/components/SearchBar.js

```diff
@@ -0,0 +1,41 @@
+//import liraries
+import React from "react";
+import { View } from "react-native";
+import { hp, wp } from "./responsive";
+import UserInput from "../constants/CutomTextInput";
+import { styles } from "../styles";
+import { color } from "react-native-elements";
+
+// create a component
+const Searchbar = ({ search, onChangeText, iconPress }) => {
+  return (
+    <View style={styles.inputCon}>
+      <View style={styles.InputMCon}>
+        <UserInput
+          backgroundColor={"white"}
+          placeholder="Tell us what you need"
+          iconName={Images.search}
+          placeholderTextColor={"#30302f"}
```

Comment:
please not use hard coded value for colors. define this color in colors file and then use it